logstreams/src/main/java/io/zeebe/distributedlog/impl/DefaultDistributedLogstreamService.java

```diff
@@ -214,28 +203,13 @@ public long append(long commitPosition, byte[] blockBuffer) {
     }
 
     final ByteBuffer buffer = ByteBuffer.wrap(blockBuffer);
-    final long appendResult = appendBlock(buffer);
-    updateCommitPosition(commitPosition);
-    return appendResult;
+    return appendBlock(commitPosition, buffer);
   }
 
-  private long appendBlock(ByteBuffer buffer) {
+  private long appendBlock(long commitPosition, ByteBuffer buffer) {
```

Comment:
I like this refactor, I think it's more intuitive and less unexpected to have the distributed log stream service use the LogStream interface and not the LogStorage. One thing is that I don't see the value in have separate append and appendBlock methods, it seems they're tied to each other and have the same purpose so maybe we can merge them.